TI-reports/2024/2024-Q4-BEST-WG.md

```diff
@@ -0,0 +1,248 @@
+- Besides a description of each coding pattern, the guide includes executable code examples for each rule, which allow for an in-depth understanding of each pattern.
+
+#### Current Status
+
+- The group is working on adding more content for a broad range of CWE rules. The status is being tracked in issue [531](https://github.com/ossf/wg-best-practices-os-developers/issues/531).
+- Dave and Bart gave a lightning talk at SOSS Community Day EU and demoed the ongoing work at the OpenSSF booth at Open Source Summit EU 2024.
+
+#### Up Next
+
+- We want to further grow the community by raising more awareness, for instance by publishing a blog post.
+- We are inviting all interested Python coders to review the current content and/or pick a new CWE rule from [531](https://github.com/ossf/wg-best-practices-os-developers/issues/531) and contribute content.
+
+
+
+
+
+### Memory Safety SIG
+
+#### Purpose
+
+- The Memory Safety SIG is a group working within the OpenSSF's Best Practices Working Group formed to understand and reduce memory safety vulnerabilities in OSS.
+
+#### Current Status
+
+- Defined the SIG's focus and terminology.
+- Collaborated with ecosystem SMEs to compile tailored memory safety best practices for both memory-safe-by-default and non-memory-safe-by-default languages.
+- Drafted the The Memory Safety Continuum document that aims to guide developers to becoming more memory safe through iterative process.
+
+#### Up Next
+
+- Release the Memory Safety Continuum document
+- Focus on interoperability and interfacing between memory safe by default and non-memory safe by default languages in software.
+
+
+
+
+
+### OpenSSF Scorecard
+
+#### Purpose
+
+- Automate analysis and trust decisions on the security posture of open source projects.
+- Use this data to proactively improve the security posture of the critical projects the world depends on.
+
+#### Current Status
+
+- Became an OpenSSF Incubating project
+- Community-led contributions for NuGet, Erlang, and Azure DevOps support
+
+#### Up Next
+
+- Make OpenSSF Scorecard (and its GitHub Action) easier to run on groups of repositories
+- Build contributor base
+- Work towards project graduation
+- Request TI funding for security audit
+- Refine project release cadence
+- Align with forthcoming OpenSSF persona work
+
+
+### Security Baseline
+
+#### Purpose
+
+- The goal of this SIG is to evolve OpenSSF security baseline for Linux Foundation wide adoption.
+- For OpenSSF adoption of the security baseline, there needs to be a home for tracking the adoption, for maintainers to raise issues to refine the security baseline, merge the baseline back to TAC lifecycle, and for OpenSSF to develop the roadmap for the security baseline. It will provide a venue for early adopters to share their reusable code and findings with other maintainers. The pilot adoption builds the foundation for wider adoption of the security baseline in OpenSSF and in Linux Foundation.
```

Comment:
What do I point a project that wants to adopt baseline to, for them to understand what they need to do?

Digging in the repo, I stumbled on https://baseline.openssf.org/, but that isn't linked from https://github.com/ossf/security-baseline or https://best.openssf.org/.

I think adopting baseline involves creating a `baseline.yml` file in my repository? We could definitely use some onboarding or getting started docs with more hand-holding!